### ISSUE TITLE
[BUG] Touching Power Button Will Cause Current App Close

### ISSUE THREAD

**Issue by People-11** [BlissRoms/bug_reports#49]
## Describe the bug
When opened an app, touching the power button (not pressing it, just touch it bcs it have fingerprint sensor ) will cause it go back to launcher.

## Steps To Reproduce
1.  Open any app
2.  Touch the power button
3.  It will go back to launcher

## Expected Behavior

```
It should nothing happend, bcs it is not a home button
```

## Current Behavior

```
Apps closed, go back to launcher.
```

## Possible Solution

```
Maybe the fingerprint sensor is mapped to the home button? Bcs some phone do have fingerprint sensor on home button.
```

## Screenshots

```
No
```

## Device Details (please complete the following information)

* Device: Zenfone 9 AI2022
* Codename: AI2022
* Build Date: Bliss-v16.6-Typhoon-OFFICIAL-gapps-20230428

**Comment by Jackeagle** [comment]
@People-11 Thank you for the report! I will reproduce it on my side and will see :)

**Comment by People-11** [comment]
This issue was resolved in 16.7.
